drake/examples/kuka_iiwa_arm/kuka_plan_runner.cc

```diff
@@ -10,7 +10,7 @@
 
 #include <lcm/lcm-cpp.hpp>
 
-#include "robotlocomotion/robot_plan_t.hpp"
+#include "robotlocomotion/robot_plan_t.hpp" // This file is not found, to be figured out with other TRIers
```

Comment:
This breaks kuka_sim. need to have a solution that works both on the robot and in sim